Description: Cope with Kubuntu setting GRUB_DISTRIBUTOR
 This is not a very good approach, and certainly not sanely upstreamable; we
 probably need to split GRUB_DISTRIBUTOR into a couple of different
 variables.
Author: Colin Watson <cjwatson@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1242417
Forwarded: not-needed
Last-Update: 2013-12-08

Index: b/util/grub-install.c
===================================================================
--- a/util/grub-install.c
+++ b/util/grub-install.c
@@ -950,6 +950,8 @@
       */
       char *t;
       efi_distributor = bootloader_id;
+      if (strcmp (efi_distributor, "kubuntu") == 0)
+	efi_distributor = "ubuntu";
       switch (platform)
 	{
 	case GRUB_INSTALL_PLATFORM_I386_EFI:
